pyproject.toml

```diff
@@ -1,6 +1,77 @@
+[project]
+name = "Flask-DebugToolbar"
+version = "0.15.1"
+description = "A toolbar overlay for debugging Flask applications."
+readme = "README.md"
+license = { file = "LICENSE.txt" }
+author = [{ name = "Michael van Tellingen" }]
+maintainers = [{ name = "Pallets Ecosystem", email = "contact@palletsprojects.com" }]
```

Comment:
This is def better than the previous matt good email since he's no longer an active maintainer, but FYI that this email doesn't go to any of the current active maintainers (me/Nick/Grey/Mac). 

So I assume this implicitly is you/the core pallets team stepping up to monitor this email?

Reply:
Eventually, I may have a way to create `flask-debugtoolbar@palletsprojects.com` as a forwarding address. For now, it's one general address. In practice, I have had maybe one legitimate email versus all the spam that this enables, so I don't think people are missing out regardless.

Reply:
My vote is let's just go with this for now, and if it starts to be a problem you can break out forwarding addresses. Like you say, probably no one will even bother to use it.